src/main/java/bio/terra/cli/service/utils/TerraCredentials.java

```diff
@@ -0,0 +1,30 @@
+package bio.terra.cli.service.utils;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.IdToken;
+
+/**
+ * Wrapper class to hold a user's credential to use when making Terra service API calls. Consists of
+ * a GoogleCredential object containing access and refresh tokens and an IdToken object containing a
+ * JWT ID token.
```

Comment:
If we have to maintain three different tokens now, how did our lives get any easier in the switch to JWT?